### ISSUE TITLE
test build failure

### ISSUE THREAD

**Issue by mihaimaruseac** [haskell/hoopl#50]
Found on stackage nightly after re-enabling the test suite

```
Building test suite 'hoopl-test' for hoopl-3.10.2.2..
[1 of 1] Compiling Main             ( testing/Main.hs, dist/build/hoopl-test/hoopl-test-tmp/Main.o )

testing/Main.hs:8:1: error:
    Could not find module ‘Test’
    Use -v to see a list of the files searched for.
  |
8 | import qualified Test
  | ^^^^^^^^^^^^^^^^^^^^^
```

Will add to expected test failures under compile fail tag.

**Comment by RyanGlScott** [comment]
This was fixed in ed178d8721bb767396c34f2ddc793399ed313623.

**Comment by mihaimaruseac** [comment]
Thanks, will wait for the next release then.
